addon/components/materialize-copyright.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+import layout from '../templates/components/materialize-copyright';
+
+export default Ember.Component.extend({
+  init: function() {
+    this._super(...arguments);
+    if (this.get('startYear') !== null) {
```

Comment:
@truenorth, this doesn't work. I only want to assert this.get('startYear') <= new Date().getFullYear(), and only if the developer decides to use the optional startYear property. Let me know if I'm misunderstanding.